### PR TITLE
Astropy dev fix

### PR DESCRIPTION
**Pull request by Cadair**
This should fix all our test fails with astropy 1.0-dev

### CODE REVIEW

**Comment by Cadair** [comment]
This is good to go and passing Travis.

**Comment by ayshih** [comment]
:+1:

**Comment by ayshih** [comment]
See #1238

**Comment by Cadair** [comment]
Closes #1238 and #1243.

**Comment by Cadair** [comment]
REVIEW ME!

**Comment by Cadair** [comment]
@rhewett If you get 5?

File: sunpy/physics/transforms/differential_rotation.py [hunk 1]
```diff
@@ -89,5 +89,6 @@ def diff_rot(ddays, latitude, rot_type='howard', frame_time='sidereal'):
 
     if frame_time == 'synodic':
         rotation_deg -= 0.9856 * delta_days
-    
-    return Longitude((np.round(rotation_deg,4)),u.deg)
+
+#    import pdb; pdb.set_trace()
```
Comment (dpshelio) [inline sunpy/physics/transforms/differential_rotation.py]: humm.... that's what you use to debug?

File: sunpy/tests/helpers.py [hunk 2]
```diff
@@ -37,3 +43,37 @@ def warnings_as_errors(request):
...
     request.addfinalizer(lambda *args: warnings.resetwarnings())
+
+
+def assert_quantity_allclose(actual, desired, rtol=1.e-7, atol=0, err_msg='', verbose=True):
+    """
+    Raise an assertion if two objects are not equal up to desired tolerance.
+
+    This is a :class:`~astropy.units.Quantity`-aware version of
+    :func:`numpy.testing.assert_allclose`.
+    """
```
Comment (dpshelio) [inline sunpy/tests/helpers.py]: Love it! :trophy: for you!

Comment (Cadair) [inline sunpy/tests/helpers.py]: Stolen from @astrofrog :)

**Comment by dpshelio** [comment]
Seems good to me :+1:

**Comment by Cadair** [comment]
I will let Travis have the final say after my rebase. 

Thanks for the review @dpshelio.

**Comment by Cadair** [comment]
Just online fails. Merging.

**Pull request merged**